### ISSUE TITLE
(Feature request) Minimize/close to tray icon

### ISSUE THREAD

**Issue by poisonborz** [zadam/trilium#3743]
### Describe feature

(Note: on Windows, but since this is similar between OSes I assume on all platforms)
While Trilium can display a tray icon, it cannot be used "persistently". Most "tray icon" apps (Slack, Signal, Ferdi, even Obsidian with plugin) have a close/minimize to tray option, where minimise/close action simply hides the window, the app can only be exited by tray context menu. This would make the tray icon 100x more useful, since currently the app will take up taskbar space regardless of the tray icon. There are utilities that can force "correct" tray behaviour like [Trayconizer](https://download.cnet.com/Trayconizer/3000-2346_4-10971946.html) but adding these options directly would be probably not much effort and would be so much cleaner. As a note, instead of "Tray" "Tray icon" would be a better/more apt name for these functionalities.

TLDR: separate options to "minimize to tray" and "close to tray", which would make the respective "_" anx "X" buttons the Window disappear, only to be revealed by clicking tray icon (or hotkey).

**Comment by supermike** [comment]
I really expect this feature, because Trayconizer is not comfortable
